test/pages/sections/items/ItemsSectionItemSpec.scala

```diff
@@ -0,0 +1,1274 @@
+          .set(ItemBulkPackagingChoicePage(testIndex1), false)
+          .set(ItemSelectPackagingPage(testIndex1, testPackagingIndex1), testPackageBag)
+          .set(ItemPackagingQuantityPage(testIndex1, testPackagingIndex1), "400")
+          .set(ItemPackagingProductTypePage(testIndex1, testPackagingIndex1), true)
+          .set(ItemPackagingSealChoicePage(testIndex1, testPackagingIndex1), false)
+        )
+
+        section.isCompleted mustBe true
+      }
+
+      "when Tobacco with fiscal marks as false and all other mandatory pages" in {
+
+        implicit val dr: DataRequest[_] = dataRequest(FakeRequest(), emptyUserAnswers
+          .set(ItemExciseProductCodePage(testIndex1), testEpcTobacco)
+          .set(ItemCommodityCodePage(testIndex1), testCnCodeTobacco)
+          .set(ItemBrandNamePage(testIndex1), ItemBrandNameModel(hasBrandName = true, Some("brand")))
+          .set(ItemCommercialDescriptionPage(testIndex1), "Petrol")
+          .set(ItemFiscalMarksChoicePage(testIndex1), false)
+          .set(ItemQuantityPage(testIndex1), BigDecimal("1000"))
+          .set(ItemNetGrossMassPage(testIndex1), ItemNetGrossMassModel(BigDecimal("2000"), BigDecimal("2105")))
+          .set(ItemBulkPackagingChoicePage(testIndex1), false)
+          .set(ItemSelectPackagingPage(testIndex1, testPackagingIndex1), testPackageBag)
+          .set(ItemPackagingQuantityPage(testIndex1, testPackagingIndex1), "400")
+          .set(ItemPackagingProductTypePage(testIndex1, testPackagingIndex1), true)
+          .set(ItemPackagingSealChoicePage(testIndex1, testPackagingIndex1), false)
+        )
+
+        section.isCompleted mustBe true
+      }
+    }
+
+    "must return false" - {
+
+      "when Tobacco and fiscal marks is true but not answer exists missing" in {
+
+        implicit val dr: DataRequest[_] = dataRequest(FakeRequest(), emptyUserAnswers
+          .set(ItemExciseProductCodePage(testIndex1), testEpcTobacco)
+          .set(ItemCommodityCodePage(testIndex1), testCnCodeTobacco)
+          .set(ItemBrandNamePage(testIndex1), ItemBrandNameModel(hasBrandName = true, Some("brand")))
+          .set(ItemCommercialDescriptionPage(testIndex1), "Petrol")
+          .set(ItemFiscalMarksChoicePage(testIndex1), true)
+          .set(ItemQuantityPage(testIndex1), BigDecimal("1000"))
+          .set(ItemNetGrossMassPage(testIndex1), ItemNetGrossMassModel(BigDecimal("2000"), BigDecimal("2105")))
+          .set(ItemBulkPackagingChoicePage(testIndex1), false)
+          .set(ItemSelectPackagingPage(testIndex1, testPackagingIndex1), testPackageBag)
+          .set(ItemPackagingQuantityPage(testIndex1, testPackagingIndex1), "400")
+          .set(ItemPackagingProductTypePage(testIndex1, testPackagingIndex1), true)
+          .set(ItemPackagingSealChoicePage(testIndex1, testPackagingIndex1), false)
+        )
+
+        section.isCompleted mustBe false
+      }
+
+      "when Tobacco and fiscal marks is missing all together" in {
```

Comment:
This looks like the same test as above?

Reply:
![image](https://github.com/hmrc/emcs-tfe-create-movement-frontend/assets/53828896/f556ea3b-475f-40dd-a228-63b6d20c77b6)